services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -8325,6 +8346,15 @@ private HelixReadWriteLiveClusterConfigRepository getReadWriteLiveClusterConfigR
     });
   }
 
+  private HelixReadWriteDarkClusterConfigRepository getReadWriteDarkClusterConfigRepository(String cluster) {
+    return clusterToDarkClusterConfigRepo.computeIfAbsent(cluster, clusterName -> {
+      HelixReadWriteDarkClusterConfigRepository clusterConfigRepository =
+          new HelixReadWriteDarkClusterConfigRepository(zkClient, adapterSerializer, clusterName);
+      clusterConfigRepository.refresh();
```

Comment:
My understanding is that we only need to do a refresh when the repository is built for the first time, basically inside the compute function. Once the repo is built, it would listen to any changes automatically; we don't need to do a hard refresh every time when the repo is retrieved.
Could you please double check?